tests/test_eventlog.py

```diff
@@ -40,12 +38,19 @@ def test_simple_databricks_log():
 
 def test_raw_databricks_log():
     event_log_path = Path("tests", "logs", "databricks.json").resolve()
+    event = get_event(event_log_path)
 
-    with tempfile.TemporaryDirectory() as temp_dir:
-        event_log = eventlog.EventLogBuilder(event_log_path.as_uri(), temp_dir).build()
+    assert all(
+        key in event
+        for key in ["Event", "Spark Version", "Timestamp", "Rollover Number", "SparkContext Id"]
+    ), "All keys are present"
 
-        with open(event_log) as log_fobj:
-            event = json.loads(log_fobj.readline())
+    assert event["Event"] == "DBCEventLoggingListenerMetadata", "Expected first event is present"
```

Comment:
is the assertion error message correct? "Expected first event is present"

Reply:
Oops, accidentally deleted that when I was having troubles at first (wrong indentation on the assertion). It's back in there now. Neal also suggested turning all the error messages to negatives ("event missing" vs ("event presenet"), which I did throughout.